config/environments/development.rb

```diff
@@ -30,6 +30,7 @@
   config.action_mailer.raise_delivery_errors = true
 
   config.action_mailer.delivery_method = :smtp
+  # config.action_mailer.smtp_settings = { :address => "localhost", :port => 1025 }
```

Comment:
What does this comment do?

Reply:
I wrote it on Slack:
"the line added in config/environments/development.rb is voluntarily commented out, we can activate that and deactivate the sendgrid settings to send emails for testing"